notebooks/en/rag_with_unstructured_data.ipynb

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
<p>brew doesn't work for every OS, does it? maybe you could provide the command other than macos or simply just redirect to installation website of both packages where people can find out the option for themselves.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>